test/e2e/framework/pv_util.go

```diff
@@ -295,53 +354,72 @@ func CreatePVsPVCs(numpvs, numpvcs int, c clientset.Interface, ns string, pvConf
 
 	// create pvs and pvcs
 	for i = 0; i < pvsToCreate; i++ {
-		pv, pvc = CreatePVPVC(c, pvConfig, pvcConfig, ns, false)
+		if pv, pvc, err = CreatePVPVC(c, pvConfig, pvcConfig, ns, false); err != nil {
+			return pvMap, pvcMap, err
```

Comment:
If for some reason the creation fails in the middle, is the caller expected to cleanup all the created resources?  Or is it this function that needs to undo all the successful creations?  Similar question for anytime we return an error from a function with partially created objects.